src/main/java/fern/network/sbml/SBMLEventHandlerObserver.java

```diff
@@ -1,128 +1,128 @@
-	 * @param net	the sbml network
-	 * @param event the event object of the sbml model
-	 */
-	public SBMLEventHandlerObserver(Simulator sim, SBMLNetwork net, Event event) throws ModelOverdeterminedException {
-		super(sim);
-		
-		this.net = net;
-		parse(event);
-	}
-	
-	private void parse(Event event) throws ModelOverdeterminedException {
-		this.name = event.getId();
-		this.trigger = new MathTree(net,
-				event.getTrigger().getMath(),
-				((SBMLPropensityCalculator)net.getPropensityCalculator()).getGlobalParameters(),
-				new HashMap<String, Double>(),
-				net.getSpeciesMapping());
-		this.delay = event.getDelay()==null ? null : new MathTree(net,
-				event.getDelay().getMath(),
-				((SBMLPropensityCalculator)net.getPropensityCalculator()).getGlobalParameters(),
-				new HashMap<String, Double>(),
-				net.getSpeciesMapping());
-		variableAssignment = new HashMap<String, MathTree>();
-		parameterAssignment = new HashMap<String, MathTree>();
-		
-		for (int i=0; i<event.getNumEventAssignments(); i++) {
-			String var = event.getEventAssignment(i).getVariable();
-			MathTree tree = new MathTree(net,
-					event.getEventAssignment(i).getMath(),
-					((SBMLPropensityCalculator)net.getPropensityCalculator()).getGlobalParameters(),
-					new HashMap<String, Double>(),
-					net.getSpeciesMapping());
-			if (net.getSpeciesMapping().containsKey(var))
-				variableAssignment.put(var, tree);
-			else
-				parameterAssignment.put(var, tree);
-		}
-	}
-	
-	private void executeEvent() {
-		for (String var : variableAssignment.keySet())
-			net.getAmountManager().setAmount(net.getSpeciesByName(var), (long)variableAssignment.get(var).calculate(net.getAmountManager(),getSimulator()));
-		for (String par : parameterAssignment.keySet()) {
-			Map<String,Double> globals = ((SBMLPropensityCalculator)net.getPropensityCalculator()).getGlobalParameters();
-			globals.put(par, parameterAssignment.get(par).calculate(net.getAmountManager(),getSimulator()));
-		}
-		getSimulator().reinitialize();
-	}
-
-	@Override
-	public void activateReaction(int mu, double tau, FireType fireType,
-			int times) {}
-
-	@Override
-	public void finished() {}
-
-	@Override
-	public void started() {
-		lastStepTriggered = true;
-	}
-
-	@Override
-	public void step() {
-	}
-	
-	@Override
-	public boolean trigger() {
-		boolean triggered = trigger.calculate(net.getAmountManager(),getSimulator())!=0;
-		if (!lastStepTriggered && triggered) {
-			lastStepTriggered = triggered;
-			double delaytime = delay==null ? 0 : delay.calculate(net.getAmountManager(),getSimulator());
-			if (delaytime<=0) 
-				executeEvent();
-			else
-				setTheta(delaytime);
-			return true;
-		}
-		lastStepTriggered = triggered;
-		return false;
-	}
-
-	@Override
-	public void theta(double theta) {
-		executeEvent();
-	}
-
-	public void setSimulatorAsync(Simulator sim) {
-		this.setSimulator(sim);
-	}
-	
-	@Override
-	public String toString() {
-		return name;
-	}
-
-}
+package fern.network.sbml;
```

Comment:
I am confused on this file. It says everything changed, but this is clearly code which existed before somewhere ?! Same with the `example/Dsmts.java` above.

Reply:
@matthiaskoenig, this happened due to different line separators in Windows (CRLF) and Ubuntu (LF). The file content remains the same only the line separators are different (as FERN is developed in Windows and I am working on Ubuntu).